src/pages/tools.en.tsx

```diff
@@ -0,0 +1,161 @@
+import React from "react";
+import { StaticQuery, graphql } from "gatsby";
+import { LocaleLink as Link } from "../components/locale-link";
+import Layout from "../components/layout";
+import { ContentfulContent, ProductList } from "./index.en";
+import { documentToReactComponents } from "@contentful/rich-text-react-renderer";
+import { ReadMoreLink } from "../components/read-more";
+import localeConfig from "../util/locale-config.json";
+import { useCurrentLocale } from "../util/use-locale";
+import PageHero from "../components/page-hero";
+
+function formatDate(dateString: string, locale?: string): string {
+  var date = new Date(dateString);
+  return date.toLocaleDateString(locale || localeConfig.DEFAULT_LOCALE, {
+    month: "short",
+    year: "numeric",
+  });
+}
+
+export const ToolsPageScaffolding = (props: ContentfulContent) => {
+  const {
+    productSectionTitle,
+    productSectionSubtitle,
+    productPageProductBlocks,
+    productIdeaBanner,
+  } = props.content;
+
+  const locale = useCurrentLocale();
+
+  return (
+    <Layout metadata={props.content.metadata}>
+      <PageHero {...props.content.pageHero} />
+
+      <ProductList
+        productSectionTitle={productSectionTitle}
+        productSectionSubtitle={productSectionSubtitle}
+        homePageProductBlocks={productPageProductBlocks}
+        productIdeaBanner={productIdeaBanner}
+      />
+
+      <div className="columns is-multiline">
+        <div className="column is-12 my-9 mb-6-mobile">
+          <h2>{props.content.pastToolsTitle}</h2>
+        </div>
+        {props.content.pastToolsBlocks.map((tool: any, i: number) => (
+          <div
+            className="column is-3 is-12-mobile pt-0 pb-10 pb-7-mobile"
+            key={i}
+          >
+            <h3 className="mb-3">{tool.toolName}</h3>
+            <div className="eyebrow is-small mb-5">
+              {formatDate(tool.toolStartDate, locale)} -{" "}
+              {formatDate(tool.toolEndDate, locale)}
+            </div>
+            <div className="title is-4 mb-5">
+              {documentToReactComponents(tool.toolDescription.json)}
+            </div>
```

Comment:
In the mocks this is h4 for desktop, but for mobile it's body-standard. We don't have h4 for mobile, so maybe we can just make it body-standard to solve this?

Reply:
Hmm, so since the mobile mocks are not final yet, I'd say let's leave this as the desktop style and we can add in mobile once it's finalized.